querybook/config/ai_assistant.yaml

```diff
@@ -0,0 +1,19 @@
+provider: ~
```

Comment:
actually, i noticed that you can also put json objects under querybook_config too, so why not put it there instead?

Reply:
because it will be read by the frontend as well, while we dont want the querybook_config exposed to frontend.